target/linux/bmips/image/bcm6368.mk

```diff
@@ -1,5 +1,21 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+define Device/actiontec_r1000h
+  $(Device/bcm63xx-cfe)
+  DEVICE_VENDOR := Actiontec
+  DEVICE_MODEL := R1000H
+  CFE_BOARD_ID := 96368VVW
+  CHIP_ID := 6368
+  IMAGES += sysupgrade.bin
+  BLOCKSIZE := 0x20000
+  FLASH_MB := 32
+  CFE_EXTRAS += --signature "$$(DEVICE_VENDOR)"
+  DEVICE_PACKAGES += $(USB2_PACKAGES) \
+    $(B43_PACKAGES) \
```

Comment:
No SPROM needed? :astonished:

Reply:
It's a detachable miniPCI card. The SPROM wasn't needed in bcm63xx for this board.